src/Parallel/dominatingset/minimal_dom_set.jl

```diff
@@ -9,6 +9,15 @@ and return the solution with the fewest vertices.
 used. This implementation is more efficient if `reps` is large.
 - If `seed >= 0`, a random generator of each process/thread is seeded with this value.
 """
-dominating_set(g::AbstractGraph{T}, reps::Integer, alg::MinimalDominatingSet; parallel=:threads, seed=-1) where T <: Integer = 
-Graphs.Parallel.generate_reduce(g, (g::AbstractGraph{T})->Graphs.dominating_set(g, alg; seed=seed), 
-(x::Vector{T}, y::Vector{T})->length(x)<length(y), reps; parallel=parallel)
+function dominating_set(
+    g::AbstractGraph{T}, reps::Integer, alg::MinimalDominatingSet;
+    parallel=:threads, kw...
```

Comment:
Would prefer if we mention `seed` and `rng` here, instead of `kw...`

Reply:
I can restore `rng=rng, seed=seed` yes, even if IMO this only leads to code complexity.

Reply:
Fixed.